cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -509,11 +512,17 @@ class DuplicateMountDirValidator(Validator):
     """
     Mount dir validator.
 
-    Verify if there are duplicated mount dirs between shared storage and ephemeral volumes.
+    Verify if there are duplicated mount dirs.
+    1. Shared storage directories can not be the same with each other.
```

Comment:
The full comment might be just: `A given shared storage mount dir can not be assigned to other shared storages or ephemeral storages.`